Tools/CISupport/ews-build/config.json

```diff
@@ -285,7 +285,7 @@
     {
       "name": "macOS-Ventura-Release-WK2-Intel-Tests-EWS", "shortname": "mac-intel-wk2", "icon": "testOnly",
       "factory": "macOSWK2Factory", "platform": "mac-ventura",
-      "configuration": "release",
+      "configuration": "release", "architectures": ["x86_64"],
```

Comment:
I'm concerned that this will cause the `download-built-product` step to fail. The architectures are present in those paths as well, and the triggering queues are indeed building both architectures.